Inconsistent major version change for org.eclipse.xtext.logging

org.eclipse.xtext.logging has for a long time been version 1.x, but the latest 2.34 build changes it to 2.x which is superficially sensible given that all other xtext plugins are 2.x. However this is a violation of https://wiki.eclipse.org/Version_Numbering 

If a breaking change is really required Xtext must go to 3.x, else please revert the common sense typo and stay at 1.x.

See https://ci.eclipse.org/ocl/job/ocl-master/864/console

[ERROR]   You requested to install 'osgi.bundle; org.eclipse.xtext.logging [1.0.0,2.0.0)' but it could not be found

I've checked and can only find explicit 1.x dependencies in the auto-generated target files, so it appears that m2e / Tycho finds the dependency for me.

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/full/full.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/full/full.parent/pom.xml
@@ -115,7 +115,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoApp/lsMavenTychoApp.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoApp/lsMavenTychoApp.parent/pom.xml
@@ -79,7 +79,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/lsMavenTychoFatjar/lsMavenTychoFatjar.parent/pom.xml
@@ -79,7 +79,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTycho/mavenTycho.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTycho/mavenTycho.parent/pom.xml
@@ -82,7 +82,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ17/mavenTychoJ17.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ17/mavenTychoJ17.parent/pom.xml
@@ -82,7 +82,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoP2/mavenTychoP2.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoP2/mavenTychoP2.parent/pom.xml
@@ -117,7 +117,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoP2J17/mavenTychoP2J17.parent/pom.xml
+++ b/org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoP2J17/mavenTychoP2J17.parent/pom.xml
@@ -117,7 +117,7 @@
 							<requirement>
 								<type>eclipse-plugin</type>
 								<id>org.eclipse.xtext.logging</id>
-								<versionRange>${xtextVersion}</versionRange>
+								<versionRange>0.0.0</versionRange>
 							</requirement>
 							<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 								platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.xtend
+++ b/org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.xtend
@@ -327,7 +327,7 @@ class ParentProjectDescriptor extends ProjectDescriptor {
 											<requirement>
 												<type>eclipse-plugin</type>
 												<id>org.eclipse.xtext.logging</id>
-												<versionRange>${xtextVersion}</versionRange>
+												<versionRange>0.0.0</versionRange>
 											</requirement>
 											<!-- to get the org.eclipse.osgi.compatibility.state plugin if the target
 												platform is Luna or later. (backward compatible with kepler and previous

--- a/org.eclipse.xtext.xtext.wizard/xtend-gen/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.java
+++ b/org.eclipse.xtext.xtext.wizard/xtend-gen/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.java
@@ -1002,7 +1002,7 @@ public class ParentProjectDescriptor extends ProjectDescriptor {
           _builder.newLine();
           _builder.append("\t");
           _builder.append("\t\t\t\t\t\t");
-          _builder.append("<versionRange>${xtextVersion}</versionRange>");
+          _builder.append("<versionRange>0.0.0</versionRange>");
           _builder.newLine();
           _builder.append("\t");
           _builder.append("\t\t\t\t\t");

